test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -45,5 +45,36 @@ public static string GetConfigContent(ServerType serverType, string iisConfig, s
 
             return content;
         }
+
+        public static string GetTargetFramework(RuntimeFlavor runtimeFlavor)
+        {
+            string targetFramework = null;
+            if (runtimeFlavor == RuntimeFlavor.Clr)
+            {
+#if NET461
+                targetFramework = "net461";
+#elif NETCOREAPP2_0 || NETCOREAPP2_1
+#else
+#error Tests targeting CLR must be compiled only on desktop.
+#endif
+            }
+            else if (runtimeFlavor == RuntimeFlavor.CoreClr)
+            {
+                targetFramework =
+#if NETCOREAPP2_0
+                "netcoreapp2.0";
+#elif NETCOREAPP2_1 || NET461
```

Comment:
Why do you check for net461 here?

Reply:
Desktop will continue to run tests for `RuntimeFlavor.CoreCLR` (I haven't opted out of those). We need to pick some TFM to run the server in this case.